src/main/java/edu/ie3/datamodel/models/TemperatureDependantLoadProfile.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models;
+
+/**
+ * Temperature dependant load profiles for night storage heating and heat pumps . The profiles rely
+ * on the VDN description for interruptable loads. For more details see <a
+ * href="https://www.bdew.de/media/documents/LPuVe-Praxisleitfaden.pdf">here</a>.
+ *
+ * <p>These are the ones determined by NBW (accessed 05/2022)
+ */
+public enum TemperatureDependantLoadProfile implements StandardLoadProfile {
```

Comment:
Compared to `BdewLoadProfile`, the parsing method `get(String key)` and `toString()` are missing. If you think that you won't need them this is totally find, just a heads up here.